src/baldr/datetime.cc

```diff
@@ -13,28 +13,59 @@
 #include "midgard/logging.h"
 #include "midgard/util.h"
 
+namespace {
+// use a cache to store already constructed sys_info's since they aren't cheap
```

Comment:
this is where the speed up is had. more info about how this works can be found here: https://github.com/HowardHinnant/date/issues/569#issue-598392857